docs/UserGuide.md

```diff
@@ -1,134 +1,121 @@
-       the file that contains the data of your previous Address Book folder.
-       
-## Command Summary
-
-Command | Format  
--------- | :-------- 
-Add | `add NAME p/PHONE_NUMBER e/EMAIL a/ADDRESS [t/TAG]...`
-Clear | `clear`
-Delete | `delete INDEX`
-Find | `find KEYWORD [MORE_KEYWORDS]`
-List | `list`
-Help | `help`
-Select | `select INDEX`
+![T-T](images/Logo.png)
+
+# **USER GUIDE** 
+---
+##**Contents**
+*   [About](https://github.com/CS2103AUG2016-T09-C3/main/blob/documents/docs/UserGuide.md#about)
+*   [Quick Start](https://github.com/CS2103AUG2016-T09-C3/main/blob/documents/docs/UserGuide.md#quick-start)
+*   [Commands](https://github.com/CS2103AUG2016-T09-C3/main/blob/documents/docs/UserGuide.md#commands)
+    -   [Add a Task](https://github.com/CS2103AUG2016-T09-C3/main/blob/documents/docs/UserGuide.md#1-add-a-task)
+    -   [Edit a Task](https://github.com/CS2103AUG2016-T09-C3/main/blob/documents/docs/UserGuide.md#2-edit-a-task)
+    -   [Delete a Task](https://github.com/CS2103AUG2016-T09-C3/main/blob/documents/docs/UserGuide.md#3-delete-a-task)
+    -   [List Tasks](https://github.com/CS2103AUG2016-T09-C3/main/blob/documents/docs/UserGuide.md#4-list-tasks)
+    -   [Help Command](https://github.com/CS2103AUG2016-T09-C3/main/blob/documents/docs/UserGuide.md#5-help-command)
+*   [T-T Bot](https://github.com/CS2103AUG2016-T09-C3/main/blob/documents/docs/UserGuide.md#t-t-bot)
+    -   [Call the T-T Bot](https://github.com/CS2103AUG2016-T09-C3/main/blob/documents/docs/UserGuide.md#1call-the-t-t-bot)
+    -   [Kill the T-T Bot](https://github.com/CS2103AUG2016-T09-C3/main/blob/documents/docs/UserGuide.md#2kill-the-t-t-bot)
+
+## **About**
+
+**Task Tracker (T-T)**, is a lightweight, easy-to-use software which helps you manage your daily tasks. It is a no-nonsense product which uses Command Line Interface to make the operations quick, easy and intuitive. 
+
+## **Quick Start**
+
+T-T is a lightweight app which does not require much time to install and run. 
+
+> Ensure you have Java version `1.8.0_60` or later installed on your PC.
+
+In order to install T-T, 
+
+1.  Open **Gitbash** at the desired location on your PC and type <br>
+    `git clone https://github.com/t09-c3/task-tracker`<br><br>
+    Press `Enter`.<br>
+    The Git respository will be cloned onto your PC.<br><br>
+
+    (*If you do not have Git on your PC, you can download the T-T files directly from https://github.com/t09-c3/task-tracker*)
+
+2.  Double Click on file named `T-T` to start the app.<br>
+
+When T-T starts up, you will be asked a friendly `"How are you feeling today?"` question. You may input the hotkey for your current mood and the colour theme for the session will change accordingly.<br>
+
+![Start Screen](images/Screen1.png)
+
+## **Commands**
+
+###1. **ADD** a Task
+
+The format for adding a new task is : <br>
+&nbsp; &nbsp;   `add [task] [date] [time] [priority]` <br>
```

Comment:
as discussed in class, it'll be good if users know whether or not changing the order of the components during input.